pychromecast/__init__.py

```diff
@@ -225,6 +226,11 @@ def volume_down(self):
         volume = round(self.status.volume_level, 1)
         return self.set_volume(volume - 0.1)
 
+    def volume_mute(self, mute):
```

Comment:
Ok, then I'll remove that volume_mute function. The other one is still needed I think (until pychromecast is refactored).